src/solve.jl

```diff
@@ -64,6 +64,48 @@ function vectorized_solve(probs, prob::ODEProblem, alg;
     ts, us
 end
 
+function vectorized_solve(probs, prob::SDEProblem, alg;
```

Comment:
throw an error if `prob.f.noise_prototype` is defined, saying it's not supported yet